pages/common/arduino.md

```diff
@@ -1,7 +1,7 @@
 # arduino
 
 > Arduino Studio - Integrated Development Environment for the Arduino platform.
-> More information: <https://github.com/arduino/Arduino/blob/master/build/shared/manpage.adoc>.
+> More information: <https://github.com/arduino/Arduino/blob/master/build/shared/manpage.adoc>, <https://manned.org/arduino>.
```

Comment:
```suggestion
> More information: <https://github.com/arduino/Arduino/blob/master/build/shared/manpage.adoc>.
```
We use manned as a fallback when official documentation doesn't exist. But here link is already present and our clients support only 1 link in this line, so I guess we can remove it.

Reply:
updated as you suggested 👍🏼